opencontrail_netns/daemon_start.py

```diff
@@ -44,12 +44,13 @@ def daemon_start():
     manager = LxcManager()
     provisioner = Provisioner(api_server=arguments.api_server,
                               api_port=arguments.api_port)
-    instance_name = '%s-%s' % (socket.gethostname(), arguments.daemon)
-    vm = provisioner.virtual_machine_locate(instance_name)
+    vrouter_name = socket.gethostname()
+    instance_name = '%s-%s' % (vrouter_name, arguments.daemon)
+    vm = provisioner.virtual_machine_locate(vrouter_name, instance_name)
```

Comment:
project should be an argument to virtual_machine_locate also.
We should do project_lookup before this call.